src/Chainweb/Cut.hs

```diff
@@ -434,7 +441,7 @@ join_ wdb prioFun a b = do
         db <- getWebBlockHeaderDb wdb cid
         (q' :> h) <- S.fold g q id $ branchDiff_ db x y
         let !h' = q' `seq` h `seq` HM.insert cid h m
-        return $! (h', q')
+        return (h', q')
```

Comment:
why remove the strictness?

Reply:
It's redundant. `(,)` is a constructor and constructor application is already in weak head normal form, so it's always strict.

Reply:
but ... strictness theater! 😄